daml-lf/validation/src/test/scala/com/digitalasset/daml/lf/validation/SerializabilitySpec.scala

```diff
@@ -319,6 +319,33 @@ class SerializabilitySpec extends AnyWordSpec with TableDrivenPropertyChecks wit
       }
 
     }
+
+    "reject unserializable interface definitions" in {
+
+      val pkg =
+        p"""
+          module NegativeTestCase {
+            interface (this: Token) = {
+              precondition False;
```

Comment:
took me a bit to figure it out but the `precondition` component turned out to be mandatory